docs/tutorial.md

```diff
@@ -0,0 +1,351 @@
+# Stackdriver Sandbox User Guide
```

Comment:
is this our existing guide or did you create a new one? This file shows as a new file.

Reply:
This is a new file. I copied the User Guide's format, but it needs to be separate because the Walkthrough Tutorial format doesn't support images (so I made changes).